### PR TITLE
ref(acl): Split Feature component into Feature / Access

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Splitting these two apart for the following reasons:

 * I'll be expanding on the Feature component very soon with some additional functionality to check hookstore for a disable component for the feature, since this functionality is specific to features I felt the responsability of the component would become biased towards feature handling with access tacked on.
 * We don't currently have any cases where we both check access and feature availability.
 * Using a component named `Feature` for ACL checks was a bit confusing

### CODE REVIEW

**Comment by billyvg** [comment]
I know we talked about this via slack, but it'd be helpful to add a PR description for others/future

File: src/sentry/static/sentry/app/components/search/sources/commandSource.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 import {createFuzzySearch} from 'app/utils/createFuzzySearch';
 import {openSudo} from 'app/actionCreators/modal';
-import Feature from 'app/components/feature';
+import Access from 'app/components/acl/access';
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: I find it a bit confusing that you import `AccessContainer` as `Access` when the `acl/access` module also defines an `Access` class.

**Comment by EvanPurkhiser** [comment]
Yeah I was just thinking after I left the office last night how I’d like to add a description to this

File: src/sentry/static/sentry/app/components/search/sources/commandSource.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: The container is simply there for wrapping the org / user context, it's an implementation detail.

Comment (markstory) [reply src/sentry/static/sentry/app/components/search/sources/commandSource.jsx]: ok.

File: src/sentry/static/sentry/app/components/acl/access.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    const hasSuperuser = !!configUser.isSuperuser;
+
+    const renderProps = {
+      hasAccess,
+      hasSuperuser,
+    };
+
+    const render = hasAccess && (!isSuperuser || hasSuperuser);
+
+    if (!render && typeof renderNoAccessMessage === 'function') {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/acl/access.jsx]: There's some duplicated logic here, but I think it's ok in order to keep this a bit simpler. We should revisit if we find ourselves adding more common code between these two components

**Review by billyvg — APPROVED** [review]
lgtm, thanks!

**Pull request merged**